ddtrace/internal/datadog/profiling/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+cmake_minimum_required(VERSION 3.19)
+
+# We're building a Python extension and expect to be called from setup.py
+set(EXTENSION_NAME "" CACHE STRING "Name of the extension")
+project(${EXTENSION_NAME})
+
+# Set C++17 as required standard for the whole project.
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+# Includes
+include(FetchContent)
+include(ExternalProject)
+include(FindPython3)
+add_subdirectory(dd_wrapper "${LIB_INSTALL_DIR}")
+
+# Find the Python interpreter
+find_package(Python3 COMPONENTS Interpreter REQUIRED)
+if (NOT Python3_INCLUDE_DIRS)
+  message(FATAL_ERROR "Python3_INCLUDE_DIRS not found")
+endif()
+
+# Cythonize the .pyx file
```

Comment:
This comment should probably go on line 28

Reply:
Indeed--thank you.  Fixed